.github/workflows/build-image.yaml

```diff
@@ -2,6 +2,7 @@
 name: Build Image
 
 on:
+  workflow_dispatch:
```

Comment:
empty? `inputs` is not required?

Reply:
I don't think so, and I really don't need one... Just have no idea why the latest merge didn't trigger this workflow?!